docs/user/neuralnet.rst

```diff
@@ -388,6 +388,83 @@ e.g. :class:`.Checkpoint`. Those should be used if you need to have
 your model saved or loaded at specific times, e.g. at the start or end
 of the training process.
 
+Input data
+----------
+
+Regular data
+^^^^^^^^^^^^
+
+skorch supports numerous input types for data. Regular input types
+that should just work are numpy arrays, torch tensors, scipy sparse
+CSR matrices, and pandas DataFrames (see also
```

Comment:
Wow, didn't know we had support for sparse matrices.

Reply:
Yes :) But note that on the batch level, we cast to sparse. The reason is that at the time of implementation, fundamental operations on sparse matrices were not supported by PyTorch, making it all but impossible to use for typical neural net stuff. I checked just now and there seems to be pretty extensive support for [sparse COO tensors](https://pytorch.org/docs/stable/sparse.html), so maybe we can revisit this again.